common.tf

```diff
@@ -168,3 +170,59 @@ module "files_table" {
     }
   ]
 }
+
+data "aws_ssm_parameter" "slack_token" {
+  name            = "/mgmt/slack/token"
```

Comment:
Should there be some sort of `${local.environment}` here instead of `mgmt` or is this fine?